chase/frontend/app/layout.tsx

```diff
@@ -1,9 +1,10 @@
 "use client";
 import "./globals.scss";
-import { Inter } from "next/font/google"; // TODO Remove Google Fonts and use local fonts (legal reasons)
+import { Inter } from "next/font/google"; // Even though Google Fonts are used – no requests are sent to Google (see NEXT.JS docs)
```

Comment:
Thats cool, can you add a link to the comment which points to the relevant parts in the docs?